pkgs/development/python-modules/manuf/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, runCommand
+, python3
+}:
+
+buildPythonPackage rec {
+  pname = "manuf";
+  version = "1.1.5";
```

```suggestion
  version = "1.1.5";
  format = "setuptools";
```
Would you please add `format`?

---

Done